src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -64,35 +61,39 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
     `;
   }
 
-  private renderStyle(): TemplateResult {
-    return html`
-      <style>
-        :host {
-          @apply --paper-font-body1;
-        }
-        ha-markdown {
-          display: block;
-          padding: 0 16px 16px;
-          -ms-user-select: initial;
-          -webkit-user-select: initial;
-          -moz-user-select: initial;
-        }
-        .markdown.no-header {
-          padding-top: 16px;
-        }
-        ha-markdown > *:first-child {
-          margin-top: 0;
-        }
-        ha-markdown > *:last-child {
-          margin-bottom: 0;
-        }
-        ha-markdown a {
-          color: var(--primary-color);
-        }
-        ha-markdown img {
-          max-width: 100%;
-        }
-      </style>
+  static get styles(): CSSResult {
+    return css`
+      :host {
+        @apply --paper-font-body1;
```

Comment:
We can remove this. This has never worked.

Reply:
`@apply` is a deprecated spec that Polymer supports, but will never become a standard.